test/fixtures/lines_stops.yml

```diff
@@ -2,3 +2,6 @@ one:
   stop_onestop_id: 1
   line_onestop_id: 1
 
+fullerton_red_line:
+  stop_onestop_id: "fullerton-id"
+  line_onestop_id: "red-line"
```

Comment:
Minor thing, but can you add a new line to the end of this file and `stops.yml`? It's a good general practice I like to follow, as not doing it can cause problems in some editors